### ISSUE TITLE
You forgot "Mirror"

### ISSUE THREAD

**Issue by Taiiwo** [khazhyk/osuapi#34]
Error when attempting to list the mods of a score using the "Mirror" mod, or at least that's what I assume.

```python
>>> score.enabled_mods
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\taiiwo\Anaconda3\lib\site-packages\osuapi\flags.py", line 68, in __repr__
    return "<%s %s>" % (type(self).__name__, " | ".join((tpl.name for tpl in self.enabled_flags)))
  File "C:\Users\taiiwo\Anaconda3\lib\site-packages\osuapi\flags.py", line 68, in <genexpr>
    return "<%s %s>" % (type(self).__name__, " | ".join((tpl.name for tpl in self.enabled_flags)))
  File "C:\Users\taiiwo\Anaconda3\lib\site-packages\osuapi\flags.py", line 88, in enabled_flags
    yield self.__flags_members__[lowest_bit]
KeyError: 1073741824
```

Fails only on:
![highscores](https://i.imgur.com/f5cc8hk.png)

Song: only my railgun (TV Size)
Difficulty: 1.25*

https://github.com/khazhyk/osuapi/blob/89a00b5e38b5742fb7b39213d94565b54fe95200/osuapi/enums.py#L24

**Comment by khazhyk** [comment]
PR's welcome!

**Comment by iotanum** [comment]
Fixed in #38

**Comment by khazhyk** [comment]
published v0.0.42
